tools/testbench/include/testbench/utils.h

```diff
@@ -52,6 +55,27 @@ struct tb_config {
 	int channels;
 	unsigned long format;
 };
+
+struct tb_ctl {
```

Comment:
Can we include the ABI header rather than duplicate ? I guess this comes from ALSA asoc.h ?

Reply:
ok, so we can do this in parallel with plugin when we ship it.